graph/dij.py

```diff
@@ -0,0 +1,66 @@
+#Dijkstra's single source shortest path algorithm
+
+import sys
+ 
+class Graph():
+
+    def __init__(self, vertices):
+        self.V = vertices
+        self.graph = [[0 for column in range(vertices)] 
+                      for row in range(vertices)]
+ 
+    def printSolution(self, dist):
+        print("Vertex \t\t Min Distance from Source")
+        for node in range(self.V):
+            print(node,"   \t\t",dist[node])
+ 
+
+    def minDistance(self, dist, MinSet):
```

Comment:
'camelCase' is not preferred for function and variable names. Use 'snake_case' like `min_distance`.